src/overlays/actors/ovl_Bg_Hidan_Dalm/z_bg_hidan_dalm.c

```diff
@@ -8,8 +8,16 @@ void BgHidanDalm_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Draw(Actor* thisx, GlobalContext* globalCtx);
+#define BgHidanDalm_WaitInteraction func_80885F4C
+void BgHidanDalm_WaitInteraction(BgHidanDalm* this, GlobalContext* globalCtx);
+void BgHidanDalm_Shrink(BgHidanDalm* this, GlobalContext* globalCtx);
+void BgHidanDalm_UpdateCollider(BgHidanDalm* this);
+
+// object
+extern Gfx D_0600BBF0[];
+extern Gfx D_0600BDF0[];
+extern UNK_TYPE D_0600DA10;
```

Comment:
I personally prefer to have the externs underneath the data, right before the functions start. Its not a hard requirement, but its what a majority of the actors in the code base do. 

I also dont think the object comment is needed, thats a given cause of segment 6 which is set up by the engine. Theyll be named to indicate that too eventually